docs/src/extend/custom-rules.md

```diff
@@ -800,6 +802,51 @@ module.exports = {
 
 To learn more about JSON Schema, we recommend looking at some examples on the [JSON Schema website](https://json-schema.org/learn/miscellaneous-examples), or reading the free [Understanding JSON Schema](https://json-schema.org/understanding-json-schema/) ebook.
 
+#### Option Defaults
```

Comment:
```suggestion
### Option Defaults
```

I think this shouldn't be under Options Schemas, as default options can be specified without a schema.